lib/karaokium_web/templates/layout/_header.html.heex

```diff
@@ -1,27 +1,98 @@
+          </svg>
+        </button>
+        <script>
+          var isClicked = false
+          function pixeis(){
+            var result = false
+            if(window.innerWidth < 700)
+              result = true
+            return result
+          }
+          function scene1()
+          {
+            document.getElementById("menu2").className = "text-center"
+            document.getElementById("content").className = "hidden"
+            document.getElementById("menu").className = "hidden"
+          }
+          function scene2()
+          {
+            document.getElementById("menu").className = "text-center"
+            document.getElementById("menu2").className = "hidden"
+            document.getElementById("content").className = "container"
+          }
+          function clickedbutton() {
+              isClicked = !isClicked
+              if(isClicked){
+                if(pixeis())
+                  scene1();
+                else
+                  scene2();
+              }
+              else{
+                document.getElementById("menu").className = "hidden"
+                document.getElementById("menu2").className = "hidden"
+                document.getElementById("content").className = "container"
+              }
+          }
+          function resized()
+          {
+            if(isClicked)
+              if(pixeis())
+                scene1()
+              else
+                scene2()
+          }
+          window.onresize = resized;
+        </script>
+      </div>
     <% end %>
-    <ul>
-      <%= for entry <- menu(@conn) do %>
-        <li>
-          <a href={entry.url}><%= entry.title %></a>
-          <%= unless Enum.empty?(entry.submenu) do %>
-            <ul>
-              <%= for subentry <- entry.submenu do %>
-                <li><a href={subentry.url}><%= subentry.title %></a></li>
-              <% end %>
-            </ul>
-          <% end %>
-        </li>
+    <div class="text-center">
+      <%= live_patch to: Routes.page_path(@conn, :index) do %>
+        <img
+          alt="Phoenix Framework Logo"
+          src={Routes.static_path(@conn, "/karaokium/images/logos/cesium-DARK.svg")}
+          height="45"
+          style="margin: 0;"
+        />
       <% end %>
-    </ul>
+    </div>
     <%= render("_user_menu.html", assigns) %>
   </nav>
 </header>
+
+<div id="menu" class="hidden">
+  <%= for entry <- menu(@conn) do %>
+    <%= unless Enum.empty?(entry.submenu) do %>
+      |
+      <%= for subentry <- entry.submenu do %>
+        <a href={subentry.url}><%= subentry.title %></a> |
+      <% end %>
+    <% end %>
+  <% end %>
+</div>
+
+<div id="menu2" class="hidden">
+  <%= for entry <- menu(@conn) do %>
+    <%= unless Enum.empty?(entry.submenu) do %>
+      <%= for subentry <- entry.submenu do %>
+        <ul>
+          <a href={subentry.url}><%= subentry.title %></a>
+        </ul>
```

Comment:
You should not be using ```<ul>``` here. If anything, it should be a ```<li>``` tag and you put the ```<ul>``` on top of the for loop
```
<ul>
  <%= for .... do %>
    <li>
      ...
    </li>
  <% end %>
</ul>